components/01-atoms/lists/_list.scss

```diff
@@ -0,0 +1,5 @@
+@mixin reset {
```

Comment:
I suggest renaming this to `list-reset` or something a bit more descriptive.

Reply:
discussed in zoom. The usage of this mixin is namespaced so the result would be `@include list.reset;` which is descriptive.